packages/ng/empty-state/empty-state-section/empty-state-section.component.ts

```diff
@@ -36,4 +36,7 @@ export class EmptyStateSectionComponent {
 		required: true,
 	})
 	description: PortalContent;
+
+	@Input()
+	hx: number = 3;
```

Comment:
```suggestion
	hx: 1 | 2 | 3 | 4 | 5 | 6 = 3;
```
If we want to be exhaustive regarding the possible values, this union type would be better (and include autocompletion for possible values)